src/commands/title-escrow/changeHolder.ts

```diff
@@ -0,0 +1,58 @@
+import { Argv } from "yargs";
+import { error, info, success, warn } from "signale";
+import { getLogger } from "../../logger";
+import { changeHolderOfTitleEscrow } from "../../implementations/title-escrow/changeHolder";
+import { TitleEscrowChangeHolderCommand } from "./title-escrow-command.type";
+import { withGasPriceOption, withNetworkAndWalletSignerOption } from "../shared";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("title-escrow:change-holder");
+
+export const command = "change-holder [options]";
+
+export const describe = "Changes the holder of the transferable record to another address";
+
+export const builder = (yargs: Argv): Argv =>
+  withGasPriceOption(
+    withNetworkAndWalletSignerOption(
+      yargs
+        .option("address", {
+          alias: "a",
+          description: "Address of the token registry that the transferable record was issued from",
+          type: "string",
+          demandOption: true,
+        })
+        .option("tokenId", {
+          description: "Hash of the transferable record",
```

Comment:
```suggestion
          description: "merkle root (document hash) of the transferable record",
```

Reply:
document batch hash :see_no_evil:

Reply:
merkle root or document hash would be more appropriate, but for now im going with what rj suggested